### ISSUE TITLE
Allele frequency sum above 1

### ISSUE THREAD

**Issue by mbdabrowska1** [HKU-BAL/Clair3#241]
In my merge_output.vcf.gz file most of the time I'm getting just a single variant per position. However I also get one position where there are two variants, but their allele frequency does no add up to 1, nor do the allelic depths add up to the total read depth for the position. Am i misunderstanding the values or is there something wrong happening with the calculations?

A snippet of my merge_output.vcf.gz looks like this:
```
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE
K03455.1	2229	.	C	A	22.46	PASS	F	GT:GQ:DP:AD:AF	1/1:22:19666:255,18257:0.9284
K03455.1	2259	.	G	A	36.43	PASS	F	GT:GQ:DP:AD:AF	1/1:36:19721:182,18951:0.9610
K03455.1	3161	.	G	A,T	18.93	PASS	F	GT:GQ:DP:AD:AF	1/2:18:20313:190,10274,17655:0.5058,0.8691
```

Where the last row is the one with multiple variants. 

Any help would be greatly appreciated

**Comment by oxygen311** [comment]
I also got same behaviour with my data for some snips:

```
chr1	1606019	.	G	A,C	10.15	PASS	F;ANN=A|intron_variant|MODIFIER|LOC105378586|LOC105378586|transcript|NR_147951.1|pseudogene|2/2|n.235-854G>A||||||,C|intron_variant|MODIFIER|LOC105378586|LOC105378586|transcript|NR_147951.1|pseudogene|2/2|n.235-854G>C||||||	GT:GQ:DP:AD:AF	1/2:10:35:5,17,30:0.4857,0.8571
```

Seems like bug to me.

**Comment by aquaskyline** [comment]
Looking into that and turn back soon

**Comment by zhengzhenxian** [comment]
@oxygen311, @mbdabrowska1,

Seem you are using amplicon data with extremely high coverage. Would you please send us a piece of reads and its reference with the given candidate included for us to pinpoint the issue? My email address is zxzheng@cs.hku.hk, thanks!

**Comment by mbdabrowska1** [comment]
I have sent the data now. Thank you for helping!

**Comment by oxygen311** [comment]
@zhengzhenxian 

Could you please confirm that @mbdabrowska1 is enough?
I'm not sure I'm authorised to share data. I definitely can say it's not Amplicon.

**Comment by zhengzhenxian** [comment]
@oxygen311 @mbdabrowska1 Thanks for the log, the AF sum bug was fixed in v1.0.5, pls kindly try the new version, thank you.
